bim2sim/sim_settings.py

```diff
@@ -548,6 +617,14 @@ def check_mandatory(self):
                     'additional 2b space boundaries.',
         for_frontend=True
     )
+    stories_to_load_guids = GuidListSetting(
+        default=[],  # Initialize with empty list by default
+        description='List of IFC GUIDs for the specific stories that should be'
+                    ' loaded. If empty, all stories will be considered '
```

Comment:
Maybe add a hint here, that loading single storeys may lead to missing ceilings (if the related slab is assigned to the storey above) and that boundary conditions may result some corrections.
Also add that this setting may be used for large buildings to reduce computational time. It is recommended to enter guids of neighboring storeys to reduce the amount of boundary condition errors.

Reply:
You are right. I added some hints to the description